src/r/inst/CITATION

```diff
@@ -0,0 +1,32 @@
+citHeader("Meyer (2019) article for the summary paper; the other one for the software implementation.")
```

Comment:
I would actually remove the second entry as this refers to the software version and it changes with the version used. Perhaps we can just include something like:

```suggestion
citHeader('To cite package "PsychroLib" in publications use:')
```